fix: correctly display status in more info

soleksy-splunk opened the pull request:
**Issue number:**

### PR Type

**What kind of change does this PR introduce?**
* [ ] Feature
* [x] Bug Fix
* [ ] Refactoring (no functional or API changes)
* [ ] Documentation Update
* [ ] Maintenance (dependency updates, CI, etc.)

## Summary

### Changes

Correctly verify status value.

Before:
Status is `true` as input is disabled: true
<img width="969" alt="Screenshot 2025-02-14 at 13 07 21" src="https://github.com/user-attachments/assets/995a4b58-b24a-40a9-93ad-1fc3eb846c70" />

Status is `false` as input is disabled: false
<img width="978" alt="Screenshot 2025-02-14 at 13 07 30" src="https://github.com/user-attachments/assets/9ef1ae33-4c61-4a1c-ad54-a82d4596cf29" />:

After:
Status is `true` as input is disabled: true
<img width="1019" alt="Screenshot 2025-02-14 at 13 16 52" src="https://github.com/user-attachments/assets/644ae013-e3eb-4d87-8b96-eaba17e6f813" />

Status is `false` as input is disabled: false
<img width="1060" alt="Screenshot 2025-02-14 at 13 16 43" src="https://github.com/user-attachments/assets/ec7bf902-70b1-4d48-a204-f745affe6a00" />


### User experience

User can see status value.

## Checklist

If an item doesn't apply to your changes, leave it unchecked.

* [x] I have performed a self-review of this change according to the [development guidelines](https://splunk.github.io/addonfactory-ucc-generator/contributing/#development-guidelines)
* [x] Tests have been added/modified to cover the changes [(testing doc)](https://splunk.github.io/addonfactory-ucc-generator/contributing/#build-and-test)
* [ ] Changes are documented
* [x] PR title and description follows the [contributing principles](https://splunk.github.io/addonfactory-ucc-generator/contributing/#pull-requests)

soleksy-splunk commented:
something to discussion, currently without mapping we display status as true and false to but is mapped like that

```
disabled: false => Status: false
disabled: true => Status: true
```

which is kind of showing reversed state.
It is fine if users use their own mapping, but without it seems unintuitive.

Should we change displayed text by default?

vtsvetkov-splunk commented:
Can you describe what has changed in user experience? Users could not see the default value without mapping?

> Should we change displayed text by default?

It is just a label in example, we don't have a default text for `disable` field, do we? I wish to change the label then

vtsvetkov-splunk commented:
I thought a bit more about this example with status and mapping. We introduced it to display and test the mapping feature, if we eliminate it here, we might decrease a test coverage.

soleksy-splunk commented:
> I thought a bit more about this example with status and mapping. We introduced it to display and test the mapping feature, if we eliminate it here, we might decrease a test coverage.

i would leave it as mapping for Status is probably most often used mapping.

Added images to show what the change is about. (Correct behaviour without mapping functionality)

What i would like to add is a default mapping (just visible) when no mapping used at all
as currently 
disabled: true  => Status: true
disabled: false => Status: false

which is kind of weird,

i would change it into:
disabled: true  => Status: Inactive
disabled: false => Status: Active

as we do it for row itself.

soleksy-splunk commented:
Also it is just something to consider but with this PR lets not modify the default displayed data.
